### PR TITLE
Refactoring unit tests from sprint 2.

### PR DESCRIPTION
**Pull request by bobloblaws93**
removed comments
addresses issue #16

### CODE REVIEW

**Comment by ghiobi** [comment]
Please read all my comments in https://github.com/Project-Epsilon/mobile/pull/16. Please do not make a new pull request when fixing issues just push the new changes. Thank you.

**Review by ghiobi — CHANGES_REQUESTED** [review]
Please read https://github.com/Project-Epsilon/mobile/pull/16. Thank you.

**Comment by ghiobi** [comment]
Please use version-2.0 as base.

File: .project [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```
Comment (ghiobi) [inline .project]: please remove file.

**Review by ghiobi — CHANGES_REQUESTED** [review]
image

File: src/providers/wallet.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+describe('Wallet Service', () => {
+    beforeEach(() => {
+       this.service = new WalletsService();
+      });
+     
+     it('Get wallets response should not be null', () => {
+        expect(this.service.getWallets()).not.toBeNull();
+      });
+  
+    });
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts]: Weird indentation

**Review by ghiobi — CHANGES_REQUESTED** [review]

File: src/providers/bank.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        expect(this.BankTransferService.deposit(2.00,'USD')).not.toBeNull();
+     });
+     
+     it('Withdrawl response should  be null', () => {
+        expect(this.BankTransferService.Withdrawl(2,'USD')).not.toBeNull();
+     });
+  
+    });
+
+  })
```
Comment (JTalarico) [inline src/providers/bank.service.spec.ts]: Indentation is all over the place in this file.

File: src/providers/currency.service.spec.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+          },
+          deps: [MockBackend, BaseRequestOptions]
+          },
+          Storage
+          MockBackend,
+          BaseRequestOptions,
+          CurrencyService
+      ]
+      
+      });
```
Comment (JTalarico) [inline src/providers/currency.service.spec.ts]: Line 15 - 30 should should be indented better. The code is hard to follow.

**Review by JTalarico — CHANGES_REQUESTED** [review]
Please fix the requested changes. Your indentation is weird.

**Review by ghiobi — APPROVED** [review]

File: src/providers/wallet.service.spec.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import { AuthHttp } from "angular2-jwt";
+import { Observable } from "rxjs";
+import { environment } from "../environments/environment";
+import{ WalletsService } from "./wallet.service";
+/**
```
Comment (ghiobi) [inline src/providers/wallet.service.spec.ts]: Why is the tsdoc so far away?

**Review by ghiobi — CHANGES_REQUESTED** [review]
Please make the changes.

**Review by JTalarico — APPROVED** [review]

File: src/providers/bank.service.spec.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+import 'rxjs/add/operator/map';
+import {environment} from "../environments/environment";
+import {BankTransferService} from './bank.service';
+import {AuthHttp} from "angular2-jwt";
+/**
+ * Bank test suite
+ */
+describe('Bank Service', () => {
+    beforeEach(() => {
+        this.BankTransferService = new BankTransferService();
```
Comment (Scieon) [inline src/providers/bank.service.spec.ts]: I seem to be getting compilation errors here, I believe it should be non-default constructor.

**Review by Scieon — CHANGES_REQUESTED** [review]

**Review by Scieon — APPROVED** [review]

**Review by ghiobi — APPROVED** [review]

**Comment by ghiobi** [comment]
Great Ronnie. We are proud of you.
